Content.Server/Atmos/EntitySystems/GasAnalyzerSystem.cs

```diff
@@ -1,46 +1,277 @@
-﻿using Content.Server.Atmos.Components;
+﻿using System.Linq;
+using Content.Server.Atmos;
+using Content.Server.Atmos.Components;
+using Content.Server.NodeContainer;
+using Content.Server.NodeContainer.Nodes;
 using Content.Server.Popups;
+using Content.Shared.Atmos;
+using Content.Shared.Atmos.Components;
 using Content.Shared.Interaction;
+using Content.Shared.Interaction.Events;
 using JetBrains.Annotations;
 using Robust.Server.GameObjects;
 using Robust.Shared.Player;
+using Robust.Shared.Utility;
+using static Content.Shared.Atmos.Components.SharedGasAnalyzerComponent;
 
 namespace Content.Server.Atmos.EntitySystems
 {
     [UsedImplicitly]
     public sealed class GasAnalyzerSystem : EntitySystem
     {
-        [Dependency] private readonly PopupSystem _popup = default!;
+        [Dependency] private readonly PopupSystem _popupSystem = default!;
```

Comment:
Don't put system in the field names as it's redundant.

Reply:
Gotcha, renamed them all